src/site/antora/modules/ROOT/pages/manual/migration.adoc

```diff
@@ -53,7 +53,10 @@ The following `log4j-core` features were moved to their own modules or were remo
 |===
 | Log4j 2 feature | Log4j 3 module
 
-| xref:manual/appenders/delegating.adoc#BlockingQueueFactory[Asynchronous appender: JCTools-based queue]
+| xref:manual/appenders/delegating.adoc#DisruptorBlockingQueueFactory[Asynchronous appender: Disruptor-based queue]
```

Comment:
There are two _Disruptor_ libraries Log4j uses: one from Conversant (the queue implementation), the other one from LMAX (the inter-thread messaging library). We can make a pass over documentation and remove this ambiguity by each time adding the appropriate context:

1. The company name (Conversant or LMAX)
2. The deliverable (queue implementation or inter-thread messaging library)
3. Both of the above

What do you think? (I can do this myself, if you prefer so.)

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/2914/commits/0fd7602c4cab3c2bcada81a93d5064dc8d2a1c13